src/ApiCompat/Microsoft.DotNet.ApiCompat.Shared/CommonResources.resx

```diff
@@ -117,12 +117,18 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="BreakingChangesFound" xml:space="preserve">
+    <value>Either fix breaking change, or update compatibility suppression files, rebuild with \p:GenerateCompatibilitySuppressions or --generate-suppression-file.</value>
```

Comment:
I would rephrase this message and start with "API compatibility differences found. If those are intentional, the APICompat baseline file can be updated by running `dotnet build /p:ApiCompatGenerateSuppressionFile=true`.

Reply:
Of course, as I'm not a native speaker, feel free to refine my suggestion.